cmd/bucket-targets.go

```diff
@@ -351,6 +351,25 @@ func (sys *BucketTargetSys) SetTarget(ctx context.Context, bucket string, tgt *m
 			return BucketRemoteTargetNotVersioned{Bucket: tgt.TargetBucket}
 		}
 	}
+
+	// Check if target is a MinIO server and alive
+	hcCtx, cancel := context.WithTimeout(ctx, 3*time.Second)
+	scheme := "http"
+	if tgt.Secure {
+		scheme = "https"
+	}
+	result := <-sys.hcClient.Alive(hcCtx, madmin.AliveOpts{}, madmin.ServerProperties{
```

Comment:
It looks like madmin.ServerProperties is not needed as well; hcClient has already the URL to ping.

Reply:
hcClient would not have been given the URL of the target replication bucket. The function did not work properly when testing without the ServerProperties